application/common/mail/get-backup-codes.html.php

```diff
@@ -0,0 +1,10 @@
+<?php
+use yii\helpers\Html as yHtml;
+
+/**
+ * @var string $displayName
```

Comment:
If it's helpful, you can copy the list of variables from one of the other emails that is already written. Every email template will get that same list of variables.

Reply:
Done